dockstore-webservice/src/main/java/io/openapi/api/impl/ToolsApiServiceImpl.java

```diff
@@ -839,8 +839,8 @@ public Response toolsIdVersionsVersionIdTypeFilesGet(String id, DescriptorType t
 
         // check for incompatible format option and header combination
         boolean zipFormat = "zip".equalsIgnoreCase(format);
-        boolean consumesHeaderJsonOnly = value.getAcceptableMediaTypes().stream().allMatch(mediaType -> mediaType.isCompatible(MediaType.APPLICATION_JSON_TYPE));
-        if (zipFormat && consumesHeaderJsonOnly) {
+        boolean jsonOnly = value.getAcceptableMediaTypes().stream().allMatch(mediaType -> mediaType.equals(MediaType.APPLICATION_JSON_TYPE));
```

Comment:
You could instead test if an application/zip compatible media type is present -- the code wouldn't need to be updated if in the unlikely event we add a 3rd representation. Alas, there is no built-in application/zip media type, so probably not worth it.

Reply:
Hmmm, will keep an eye on it, but agreed on not now